connect/src/main/java/com/stripe/android/connect/EmbeddedComponentManager.kt

```diff
@@ -21,17 +35,31 @@ class EmbeddedComponentManager(
     private val fetchClientSecretCallback: FetchClientSecretCallback,
     appearance: Appearance = Appearance(),
     private val customFonts: List<CustomFontSource> = emptyList(),
+    private val logger: Logger = Logger.getInstance(enableLogging = BuildConfig.DEBUG),
+    private val isDebugBuild: Boolean = BuildConfig.DEBUG,
```

Comment:
We shouldn't add this to the public API, which `EmbeddedComponentManager` and this constructor will be part of (we should remove the `@RestrictTo` above). I do think we should add a `debug: Boolean` property to `Configuration`, but `logger` should not be exposed and instead should be instantiated internally.